src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -395,12 +395,18 @@ export const PublishingAccordion = ({
                     : t('registration.public_page.tasks_panel.registration_is_published')}
                 </Typography>
               </>
-            ) : (
+            ) : registrationHasFile ? (
               <Trans
                 t={t}
                 i18nKey="registration.public_page.tasks_panel.registration_is_published_workflow2"
                 components={[<Typography paragraph key="1" />]}
               />
+            ) : (
+              <Trans
+                t={t}
+                i18nKey="registration.public_page.tasks_panel.registration_is_published_workflow2_without_file"
+                components={[<Typography paragraph key="1" />]}
+              />
             )}
```

Comment:
ville aller helst hatt ternary inni `<Trans>` her, for å unngå en del ekstra kode:
```html
<Trans
  t={t}
  i18nKey={
    registrationHasFile
      ? 'registration.public_page.tasks_panel.registration_is_published_workflow2'
      : 'registration.public_page.tasks_panel.registration_is_published_workflow2_without_file'
  }
  components={[<Typography paragraph key="1" />]}
/>```

Men det kopliserer sjekking for ubrukte i18n-keys en del, så tenker det er enklest å bare splitte i to ulike `<Trans>` i stedet